agent/taskresource/envFiles/envfile.go

```diff
@@ -358,9 +360,14 @@ func (envfile *EnvironmentFileResource) writeEnvFile(writeFunc func(file oswrapp
 	return nil
 }
 
-// Cleanup removes env files downloaded for the task
+// Cleanup removes env file directory for the task
 func (envfile *EnvironmentFileResource) Cleanup() error {
-	// TODO
+	err := envfile.os.RemoveAll(envfile.resourceDir)
```

Comment:
how about force removing it if removal fails due to some reason?

Reply:
There's no force option available. We do the same firelens resource dir. I can add retry if this is concern?

Reply:
ok, we can add retries if this causes issues in the future.